frameworklauncher/doc/USERMANUAL.md

```diff
@@ -573,14 +573,51 @@ Launcher sets up below EnvironmentVariables for each User Service to use:
 |:---- |:---- |
 | APP_ID | Framework's current associated Application ID. |
 | CONTAINER_ID | Task's current associated Container ID. |
+| CONTAINER_IP | Task's current associated Container IP address. |
 
-3. Used to get the assigned Resource, only set when corresponding feature is enabled.
+3. Used to access the Launcher managed public HDFS store.
```

Comment:
>Used to [](start = 3, length = 7)

Used to  -> To ?

Reply:
This is align with others 

---
In reply to: [188147033](https://github.com/Microsoft/pai/pull/575#discussion_r188147033) [](ancestors = 188147033)